core/cap-0038.md

```diff
@@ -923,10 +925,12 @@ if isAuthorizedToMaintainLiabilities(tl) && !isAuthorizedToMaintainLiabilities(e
             cb.asset = assetAndAmount.asset
             cb.claimant = trustor
 
-            if isClawbackEnabledOnTrustline(tl)
+            if isClawbackEnabledOnTrustline(tlA)
                 cb.flags = CLAIMABLE_BALANCE_CLAWBACK_ENABLED_FLAG
 
-            create(cb)
+            // will return false if cbSponsoringAcc does not have the appropriate reserves
+            if(!create(cb))
+                Fail with ALLOW_TRUST_CANT_REVOKE or SET_TRUST_LINE_FLAGS_CANT_REVOKE
```

Comment:
I think this is going to get confusing. Right now we use `CANT_REVOKE` to indicate that `AUTH_REVOCABLE` isn't set. We should probably add a new error code like `LOW_RESERVE`, what do you think?

Reply:
Yeah I think that makes sense